content/en/tracing/guide/configuring-primary-operation.md

```diff
@@ -82,15 +82,128 @@ By default, the resource name would be set to this as it’s the name of the fun
 
 ## OpenTracing
 
-When using Datadog, the Opentracing operation name is a resource and the Opentracing "component" tag is Datadog's span name. To define (in Opentracing terms) a span that has the resource "/user/profile", and the span name "http.request", use this Go example:
+When using Datadog, the OpenTracing operation name is a resource and the OpenTracing "component" tag is Datadog's span name. For example, to define (in OpenTracing terms) a span that has the resource "/user/profile", and the span name "http.request":
 
-```text
+{{< programming-lang-wrapper langs="java,python,ruby,go,nodejs,.NET,php" >}}
+{{< programming-lang lang="java" >}}
+
+
+
+```java
+Span span = tracer.buildSpan("http.request").start();
+
+try (Scope scope = tracer.activateSpan(span)) {
+    span.setTag("service.name", "service_name");
+    span.setTag("resource.name", "/user/profile");
+    // code being traced
+} finally {
+    span.finish();
+}
+
+```
+
+For more information on how to set up Java and OpenTracing, see this [article][3].
+
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+
+```python
+  from ddtrace.opentracer.tags import Tags
+  import opentracing
+  span = opentracing.tracer.start_span('http.request')
+  span.set_tag(Tags.RESOURCE_NAME, '/user/profile')
+  span.set_tag(Tags.SPAN_TYPE, 'web')
+
+  # ...
+  span.finish()
+
+```
+
+For more information on how to set up Python and OpenTracing, see this [article][4].
+
+{{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+
+```ruby
+  OpenTracing.start_active_span('http.request') do |scope|
+    scope.span.datadog_span.resource = '/user/profile'
+    # code being traced
+  end
+```
+For more information on how to set up Ruby and OpenTracing, see this [article][5].
+
+{{< /programming-lang >}}
+{{< programming-lang lang="go" >}}
+
+
+```go
 opentracing.StartSpan("http.request", opentracer.ResourceName("/user/profile"))
 ```
 
+For more information on how to set up Go and OpenTracing, see this [article][6].
+
+{{< /programming-lang >}}
+{{< programming-lang lang="nodejs" >}}
+
+
+```javascript
+  const span = tracer.startSpan('http.request');
+  span.setTag('resource.name',  ‘/user/profile’)
+  span.setTag('span.type', 'web')
+// code being traced
+  span.finish();
+```
+
+For more information on how to set up Node.js and OpenTracing, refer to this [article][7].
+
+{{< /programming-lang >}}
+{{< programming-lang lang=".NET" >}}
+
+
+```csharp
+using OpenTracing;
+using OpenTracing.Util;
+
+using (IScope scope = GlobalTracer.Instance.BuildSpan("http.request").StartActive(finishSpanOnDispose: true))
```

Comment:
[Nit] better to use `var` instead of `IScope`. It's not a big deal in that case but in some cases it could reduce changes to be done on customer side if we we were to change the return type.

`using (var scope = GlobalTracer.Instance.BuildSpan("http.request").StartActive(finishSpanOnDispose: true))`

Reply:
Thanks! I tested out `var scope` and got the same result as using `IScope scope`. This should be resolved with my latest commit, so the example is now:

```
using (var scope = GlobalTracer.Instance.BuildSpan("http.request").StartActive(finishSpanOnDispose: true))
{
    scope.Span.SetTag("resource.name", "/user/profile");
    // code being traced
}
```